### ISSUE TITLE
Disabling the watercooler doesn't actually disable it...

### ISSUE THREAD

**Issue by brianlovin** [withspectrum/spectrum#4862]
If you disable the watercooler, the thread still exists. It just won't necessarily be pinned at the top of the feed. To repro: create a new community, enable the watercooler, disable the watercooler, go to your community profile. You'll see the watercooler thread there.

**Comment by brianlovin** [comment]
I'm not sure the solution here, I know that the profile redesign will help a lot, but if you have open chat disabled we gotta make sure that thread doesn't show up in any feed.

**Comment by mxstbr** [comment]
Yes that is on purpose, I doubt that's a very common case. If anything, we should rename the thread to be "Open Chat" so it's clearer what it's about, which will likely cause fewer people to disable it.

I vote for deleting it manually for folks who disable it.

**Comment by brianlovin** [comment]
I’m fine deleting it but should they be able to restore it?

**Comment by dev-drprasad** [comment]
@brianlovin what is **watercooler** ? i searched in google, but you know the result

**Comment by brianlovin** [comment]
@dev-drprasad it's an open chat thread like https://spectrum.chat/react/general/react-watercooler~257957e0-ef20-4757-b25e-82156b89131a

**Comment by lookapanda** [comment]
I don't think the watercooler should be deleted by disabling the watercooler option. It should only be deletable, if one actually deletes it (which could cause a new prompt to create one).

See it like this: One may want to have a temporary watercooler. They start it up, people chat in it. After some while, they may want to close it again but they want to repeat this in the future and they don't want to lose what's already been said in the chat. If that makes sense?

**Comment by mxstbr** [comment]
@lookapanda exactly, that's why I implemented it the way I implemented it! I also don't think we should hide the thread, maybe it's just a matter of removing the "Watercooler" indicator?
